umbra-js/src/classes/Umbra.ts

```diff
@@ -391,7 +397,25 @@ export class Umbra {
   }
 
   /**
-   * @notice Fetches all Umbra event logs using The Graph
+   * @notice Fetches Umbra event logs starting from the block user registered their stealth keys in using Goldsky, if available, falling back to RPC if not
+   * @param overrides Override the start and end block used for scanning; ignored if using Goldsky
+   * @returns A list of Announcement events supplemented with additional metadata, such as the sender, block,
+   * timestamp, and txhash
+   */
+  async fetchSomeAnnouncements(
+    Signer: JsonRpcSigner,
+    address: string,
+    overrides: ScanOverrides = {}
+  ): Promise<AnnouncementDetail[]> {
+    const registeredBlockNumebr = await getBlockNumberUserRegistered(address, 0, Signer);
```

Comment:
`registeredBlockNumber` instead of `registeredBlockNumebr`

Reply:
nice catch fixed https://github.com/ScopeLift/umbra-protocol/pull/549/commits/5c9d0b6f28897cdd7b868c03da62604dfb1d6f03